pom.xml

```diff
@@ -63,6 +64,29 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
```

Comment:
@rotty3000 Can you please create a PR with a short comment explaining why we need this in dependencyManagement. Thanks.

Reply:
Yes, will do!